cockatrice/src/settings/shortcuts_settings.h

```diff
@@ -536,7 +536,7 @@ class ShortcutsSettings : public QObject
                                                 parseSequenceString(""),
                                                 ShortcutGroup::Move_bottom)},
         {"Player/aDrawArrow", ShortcutKey(QT_TRANSLATE_NOOP("shortcutsTab", "Draw Arrow..."),
-                                          parseSequenceString("Ctrl+A"),
+                                          parseSequenceString(""),
```

Comment:
@ebbit1q Is this a concern to you?

Reply:
yes, while most people might draw arrows with the right mouse drag we want to offer a shortcut as well, some people might prefer it and some people might have disabilities that make it easier to use a single press instead of a drag with the mouse.